src/pages/worker/Worker.jsx

```diff
@@ -1,4 +1,7 @@
-export const Worker = ({ worker }) => {
+import { Link } from "react-router-dom";
+
+export const Worker = (props) => {
+  const { worker } = props;
   return (
     <tbody>
```

Comment:
`tbody` element should be defined only once for all worker rows in a table. Please move tbody to parent component (`WorkersList`). After that please rename this component to `WorkerRow`